src/components/Button/Button.js

```diff
@@ -4,13 +4,13 @@ import { useTheme } from '@material-ui/core';
 
 import StyledButton from './Style';
 
-export default function CommonButton({ children, color }) {
+export default function CommonButton(props) {
   const theme = useTheme();
 
   // One option to get theme and supply it to styled compoenents
   return (
-    <StyledButton color={color} theme={theme} variant="contained">
-      {children}
+    <StyledButton {...props} color="primary" theme={theme} variant="contained">
```

Comment:
Why have you removed the color prop and hard code the color value it is a common component so make color dynamic based on it's props